src/site/stages/build/index.js

```diff
@@ -52,6 +52,7 @@ function defaultBuild(BUILD_OPTIONS) {
   smith.use(filenames());
 
   smith.use(applyFragments(BUILD_OPTIONS));
+  smith.use(checkCollections(BUILD_OPTIONS));
```

Comment:
Just wondering, is there a specific reason you chose to register this plugin at this point in the plugin chain?